crimsobot/utils/games.py

```diff
@@ -13,6 +14,40 @@
 DiscordUser = Union[discord.User, discord.Member]
 
 
+def return_cball_answer(ctx: commands.Context) -> str:  # function to give first answer a ctx to work with
+    # don't know if this is any better than just putting it
+    # inside of the crimsoball command
+    answer_list = [
+            '{} haha ping'.format(ctx.message.author.mention),
+            'ye!',
+            '**no**',
+            'what do you think?',
+            '*perhaps*',
+            'OMAN',
+            "i can't answer this, you need an adult",
+            'absolutely!\n\n\n`not`',
+            'of course!',
+            'according to quantum superposition, the answer was both yes and no before you asked.',
+            "is the sky blue?\n\n(is it? i don't know. i don't have eyes.)",
+            "i can't be bothered with this right now.",
+            'funny you should ask--',
+            'fine, sure, whatever',
+            '<:xok:551174281367650356>',
+            'ask seannerz. ping him now and ask.',
+            'ehhhh sure',
+            'hmmmm. no.',
+            'uhhhhhhhhh',
+            '<:uhhhh:495249068789071882>',
+            'eat glass!',
+            'it is important that you stop bothering me.',
+            'you CANNOT be serious',
+            'sure? how would i know?',
+            'what heck',
+            'random_response',  # leave this alone
+        ]
+    return random.choice(answer_list)
```

Comment:
There should (potentially) be a newline before the return statement since that seems to be the formatting chosen the majority of the time for this project - though not all the time. Is the seeming inconsistency intentional @williammck?

Reply:
Fixed with a4c8767 as it's the majority, and generally looks better